src/resources/elements/primeDesignSystem/demos/pCountdownCircularDemo.html

```diff
@@ -0,0 +1,74 @@
+<template>
+  <style>
+    h1 {
+      font-size: 20px;
+      margin: 36px 0 20px 0;
+    }
+
+    .circularDemoContent {
+      display: inline-flex;
+      flex-direction: column;
+      row-gap: 20px;
+      margin-left: 20px;
+      align-items: center;
+      padding: 24px;
+      border-radius: 10px;
+      /* background-color: #403453; is supposed to be this but doesn't work */
+    }
+
+    .ticked {
+      display: inline-block;
+    }
+
+    .clickable {
+      cursor: pointer;
+    }
+  </style>
+  <h1>Countdown from 10 seconds, alert on stopped</h1>
+  <div class="circularDemoContent">
```

Comment:
Not sure if it's a priority right now but the circular border looks jagged on Chrome when it's running. Should we try to fix it?

Also, not sure if it's a priority but should we animate the "ticks" between numbers? Right now it jumps from a number to another. It doesn't look that good when having a 3 second countdown for example.

Reply:
I would like to add animation, and have spent time on it, but have not been able to figure out how to do it, for either text or the curvy line.  Since it is not required in the mock, I don't think we should spend more time on it ATM.